src/tls13formats.rs

```diff
@@ -601,14 +620,13 @@ pub fn get_handshake_messages4(
 }
 
 pub fn find_handshake_message(ty: HandshakeType, payload: &HandshakeData, start: usize) -> bool {
-    let HandshakeData(p) = payload;
-    if p.len() < start + 4 {
+    if payload.0.len() < start + 4 {
```

Comment:
is `payload.0` preferable to pattern-matching `HandshakeData`?

Reply:
Code I touch, I make more rusty.
Deconstructing tuple structs adds visual noise and makes it less readable. Generally, tuple structs are used by index. But ideally we change all structs to named structs to have the code more readable. I'm doing that along the way as well, when touching the struct. Looks like I didn't touch `HandshakeData` yet.